app-text/libgepub/libgepub-0.7.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org meson xdg
+
+DESCRIPTION="GObject based library for handling and rendering epub documents"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/libgepub"
+
+LICENSE="LGPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="-widget +introspection"
```

Comment:
USE flags should be in alphabetical order, but more importantly, you cannot do a `-` USE flag. That's just the default. See the QA report:



0.7.0 | BadDefaultUseFlags | bad default USE flag: '-widget'
-- | -- | --

I'm going to change the name to `webkit` since that's a commonly used name for flags that depend on webkit-gtk.